client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -55,9 +107,9 @@ public void testDeleteIndex() throws IOException {
     }
 
     private static void createIndex(String index) throws IOException {
-        Response response = client().performRequest("PUT", index);
+        CreateIndexResponse response = highLevelClient().indices().createIndex(new CreateIndexRequest(index));
 
-        assertEquals(200, response.getStatusLine().getStatusCode());
+        assertTrue(response.isAcknowledged());
```

Comment:
Would obviously be trivial to revert to using the low-level client here, but I'd be wary of applying that principle later on - e.g., it we're testing Get Alias, it would be a lot more costly to create the test data using the low-level client, rather than the high-level client Create Alias; and so forth.